src/SprykerEco/Yves/AmazonPay/Theme/default/components/atoms/amazon-submit-confirmation-button/amazon-submit-confirmation-button.ts

```diff
@@ -0,0 +1,86 @@
+import Component from 'ShopUi/models/component';
+
+declare const window: {
+    OffAmazonPayments: {
+        initConfirmationFlow(sellerId: string, orderId: string, callback: (confirmationFlow: ConfirmationFlow) => void): void
+    }
+};
+
+interface ConfirmationFlow {
+    success(): void;
+    error(): void;
+}
+
+const XHR_SUCCESS_CODE = 200;
+
+export default class AmazonSubmitConfirmationButton extends Component {
+    protected button: HTMLButtonElement;
+    protected xhr: XMLHttpRequest;
+
+    constructor() {
+        super();
+        this.xhr = new XMLHttpRequest();
+    }
+
+    protected readyCallback(): void {}
+
+    mountCallback(): void {
+        this.button = this.querySelector(`.${this.jsName}__button`);
```

Comment:
You can use `getElementsByClassName` in this case.

Reply:
fixed